element-connector/src/test/java/org/eclipse/californium/elements/rule/ThreadsRule.java

```diff
@@ -182,7 +182,7 @@ public void checkThreadLeak(List<Thread> activeThreads, boolean reportLeakAsExce
 	 * @param list list of threads
 	 */
 	public void dump(String message, List<Thread> list) {
-		LOGGER.info("Threads {}: {} threads", message, list.size());
+		LOGGER.trace("Threads {}: {} threads", message, list.size());
```

Comment:
If a logging statement is the first statement in the method body, should we consider `method name`? We can make a set of keywords for method names.